samples/kvs_gstreamer_multistream_sample.cpp

```diff
@@ -263,6 +327,22 @@ static GstFlowReturn on_new_sample(GstElement *sink, CustomData *data) {
                                std::chrono::nanoseconds(buffer->dts), kinesis_video_flags)) {
             GST_WARNING("Dropped frame");
         }
+        
+        // Add AWS_KINESISVIDEO_IMAGE_GENERATION metadata every 5 seconds
+        auto current_time = std::chrono::system_clock::now();
+        auto time_since_epoch = std::chrono::duration_cast<std::chrono::seconds>(current_time.time_since_epoch()).count();
+        
+        if (time_since_epoch - data->last_metadata_time[stream_handle_key] >= 5) {
+            auto kvs_stream = data->kinesis_video_stream_handles[stream_handle_key];
+            bool metadata_result = kvs_stream->putFragmentMetadata("AWS_KINESISVIDEO_IMAGE_GENERATION", "", false);
```

Comment:
This one you should use `putEventMetadata` (example in the [other PR](https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-cpp/pull/1176)) since `putFragmentMetadata` will return an error if the key starts with `AWS_`.

Reply:
@sirknightj thank you! This has helped. I am now able to add the tags to video streams.